From bc0efc857b5708ea572e3a4c57c9689cf48c50ad Mon Sep 17 00:00:00 2001
From: Brian Dolbec <dolsen@gentoo.org>
Date: Sat, 15 Dec 2012 20:48:17 -0800
Subject: [PATCH] Fix setup.py to use PVR instead of VERSION from the
 environment so that egg-info is set correctly during the
 merge.

---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 8a9f980..7a53076 100755
--- a/setup.py
+++ b/setup.py
@@ -12,7 +12,7 @@ import os
 import io


-__version__ = os.getenv('VERSION', default='9999')
+__version__ = os.getenv('VERSION', default=os.getenv('PVR', default='9999'))

 cwd = os.getcwd()

--
1.8.0

